### DISCUSSION TITLE
Unreliable notifications

### DISCUSSION THREAD

**Discussion by woj-tek** [iSoron/uhabits#946] [General Discussion]
I'm not *sure* if that's the case, but I think that sometimes notifications "disappear". In general I seem to see all of them, but sometimes when I open the app I see that some entries are still with question mark and that's rather unusual - I have permanent notifications so I can't dismiss them. I have to select either "yes" or "no" on each habit. Having question marks would indicated that the notification "disappears". 

It was working very consistent before version 2. I'm not sure if anything changed or if anyone else noticed something similar?

**Comment by Aldis97** [comment]
Oh I experience the similar thing, more or less rn., now waiting second reply.

**Comment by baincd** [comment]
I've been experiencing the same issue.  The notifications do notify on time, but after a while (maybe an hour or a couple of hours) the notifications disappear.  This seems to have started with version 2.  I don't recall this issue (at least with any sort of frequency) in version 1.

**Reply by baincd** [reply]
I had an interesting behavior recently, that I want to share in case this helps shed some light into why notifications disappear.

I had several habit notifications (at least 3).  I clicked "Yes" in one of the notifications, but the notification did not go away as usual.  I clicked "Yes" several more times with the same result.

So I opened the Habits app, and from the main screen I long-pressed the checkmark I wanted to check.  Then the habits app closed and all the habit notifications disappeared.  When I opened the Habits app again, the habit was still not checked.

(This is from memory, so hopefully I didn't misremember or forget any details.  I will update if I experience this again).
